src/convertToObject.js

```diff
@@ -6,7 +6,20 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const styles = sourceString
+    .split(';')
+    .map((style) => style.split(':'))
+    .reduce((acc, [key, value]) => {
+      if ((key, value)) {
```

Comment:
[CODE KNOWLEDGE] - The condition in the if statement seems to be incorrect due to a misplaced comma. It appears you intended to check if both key and value are truthy, but the current condition only evaluates the truthiness of the value because of the comma operator. Instead, you should use a logical operator like `&&` to ensure both key and value are truthy before proceeding with the trimming and assignment.